sdk/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced-mlflow.ipynb

```diff
@@ -219,7 +220,7 @@
     "### Using lags and rolling window features\n",
     "This training is also using the **target lags**, that is the previous values of the target variables, meaning the prediction uses a horizon. We therefore must still specify the `forecast_horizon` that the model will learn to forecast. The `target_lags` keyword specifies how far back we will construct the lags of the target variable, and the `target_rolling_window_size` specifies the size of the rolling window over which we will generate the `max`, `min` and `sum` features.\n",
     "\n",
-    "This notebook uses the .set_training(blocked_models=) parameter to exclude some models that take a longer time to train on this dataset.  You can choose to remove models from the blocked_models list but you may need to increase the iteration_timeout_minutes parameter value to get results."
+    "This notebook uses the .set_training(blocked_training_algorithms=) parameter to exclude some models that take a longer time to train on this dataset.  You can choose to remove models from the blocked_training_algorithms list but you may need to increase the iteration_timeout_minutes parameter value to get results."
```

Comment:
.set_training(blocked_training_algorithms=) -> remove '=' at the end?

iteration_timeout_minutes -> trial_timeout_minutes

(also in automl-forecasting-task-energy-demand-advanced.ipynb)